src/app/services/transcription/transcription.service.spec.ts

```diff
@@ -69,6 +70,25 @@ describe('TranscriptionService', () => {
     });
   });
 
+  describe('#getWorkRequests', () => {
+    it('should transform hearing dates', (done) => {
+      const mockRequests = [
+        { transcription_id: 1, hearing_date: '2023-11-01' },
+        { transcription_id: 2, hearing_date: '2023-11-02' },
+      ] as Partial<WorkRequest>;
+
+      service.getWorkRequests().subscribe((transformedRequests) => {
+        expect(transformedRequests[0].hearing_date).toBe('2023-11-01T00:00:00Z');
+        expect(transformedRequests[1].hearing_date).toBe('2023-11-02T00:00:00Z');
+
+        done();
```

Comment:
Does having this here not finish the test and potentially ignore the lines below https://github.com/hmcts/darts-portal/pull/595/files#diff-5c1411b5ebd6e7a028429d102c6c28585ceacdf25df1431d71fe7b8bfd292785R87-R88?

Reply:
I retested making some slight changes and can confirm that the test fails if that URL is not as expected so it doesn't appear to be ignoring them